daphne/src/auth.rs

```diff
@@ -14,7 +14,7 @@ use crate::{
 use async_trait::async_trait;
 use serde::{Deserialize, Serialize};
 
-/// A bearer token used for authorizing DAP requests as specified in draft-ietf-ppm-dap-02.
+/// A bearer token used for authorizing DAP requests as specified in draft-ietf-ppm-dap-03.
```

Comment:
Hmm, actually this comment is a bit out-of-date. The bearer token thing goes back to draft-01, but is no longer specified in the current draft.